shared/admin-tools/util.ts

```diff
@@ -88,57 +112,6 @@ export const getClientId = async (
   return undefined;
 };
 
-// Generate a strong password that makes Cognito happy
-export const generatePassword = (numChars: number): string => {
```

Comment:
We implemented this because it gave us control over which special characters to include. Before implementing this, we kept getting passwords with characters that our various bash scripts had trouble with.